ui/lib/core/addon/components/form-field.hbs

```diff
@@ -6,7 +6,42 @@
 {{! template-lint-configure simple-unless "warn" }}
 <div class="field" data-test-field={{or @attr.name true}}>
   {{#if this.isHdsFormField}}
-    {{! TODO the template logic for the form fields migrated to HDS will appear here }}
+    {{#if @attr.options.possibleValues}}
+      {{#if (eq @attr.options.editType "checkboxList")}}
+        <Hds::Form::Checkbox::Group @name={{@attr.name}} data-test-input={{@attr.name}} as |G|>
+          {{#if this.labelString}}
+            <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
+          {{/if}}
+          {{#if this.showHelpText}}
```

Comment:
This getter doesn't account for checking if there's helpText, I wonder if we should update that there so this div doesn't render unless helpText exists (instead of having to add it to every conditional wrapping helper text)

Reply:
@hellobontempo what do you prefer? I can update the getter in this PR, or I can create a separate PR for this (no biggie, I just want to follow your existing processes).

Reply:
Looking at the code again, it looks like there is some additional cleanup worth tackling all at once. (Since `FormFieldLabel` relies on the boolean to pass along the `helpText` or not). If you want to tackle all of that now, a separate PR may be best. Though I think adding the second check here ` {{#if (and this.showHelpText @attr.options.helpText)}}` might be smoothest for now and that tech cleanup can be postponed until the end. What do you think?

Reply:
If you look at this PR https://github.com/hashicorp/vault/pull/29960/ you will see that the code is being duplicated [here](https://github.com/hashicorp/vault/blob/d7109460ff93f61223182fef07fde7a20319343a/ui/lib/core/addon/components/form-field.hbs#L12-L26) and [here](https://github.com/hashicorp/vault/blob/d7109460ff93f61223182fef07fde7a20319343a/ui/lib/core/addon/components/form-field.hbs#L52-L65) (and will be duplicated even more, the more we migrate components here; I am going to think how to abstract this "thing" once we have a few more instances, so for now we have to live with that duplication)

All to say that probably is better to do this at getter level, already (now). I am going to open a PR for this, as parallel stream of work.

Reply:
Great! I agree (just didn't want to disrupt your workflow 😄 ) 
Consolidating the getter to be something like this probably makes sense, and removing all references `@attr.options.helpText` to but I'll defer to you since you're in the thick of all that logic now.
```js
  get showHelpText() {
    const { options } = this.args.attr;
    if (this.args.showHelpText !== false && options.helpText) return options.helpText;
    return '';
  }